packages/wrangler/src/versions/deployments/view.ts

```diff
@@ -0,0 +1,41 @@
+import { UserError } from "../../errors";
+import { printWranglerBanner } from "../../update-check";
+import type {
+	CommonYargsArgv,
+	StrictYargsOptionsToInterface,
+} from "../../yargs-types";
+
+export type VersionsDeploymentsViewArgs = StrictYargsOptionsToInterface<
+	typeof versionsDeploymentsViewOptions
+>;
+
+export function versionsDeploymentsViewOptions(yargs: CommonYargsArgv) {
+	return yargs
+		.option("name", {
+			describe: "Name of the worker",
+			type: "string",
+			requiresArg: true,
+		})
+		.positional("deployment-id", {
+			describe:
+				"Deprecated. Deployment ID is now referred to as Version ID. Please use `wrangler versions view [version-id]` instead.",
+			type: "string",
+			requiresArg: true,
+		});
+}
+
+export async function versionsDeploymentsViewHandler(
+	args: VersionsDeploymentsViewArgs
+) {
+	await printWranglerBanner();
+
+	if (args.deploymentId === undefined) {
+		throw new UserError(
+			"`wrangler deployments view` is deprecated and will be removed in a future major version. Please use `wrangler deployments status --x-versions` instead."
+		);
+	} else {
+		throw new UserError(
+			"`wrangler deployments view <deployment-id>` is deprecated and will be removed in a future major version. Deployment ID is now referred to as Version ID. Please use `wrangler versions view [version-id] --x-versions` instead."
+		);
```

Comment:
@petebacondarwin how about this instead?

```suggestion
	if (args.deploymentId === undefined) {
		throw new UserError(
			"`wrangler deployments view` has been renamed `wrangler deployments status --x-versions`. Please use that command instead."
		);
	} else {
		throw new UserError(
			"`wrangler deployments view <deployment-id>` has been renamed `wrangler versions view [version-id] --x-versions`. Please use that command instead."
		);
	}
```

Reply:
LGTM!